django/core/validators.py

```diff
@@ -204,9 +204,9 @@ def __init__(self, message=None, code=None, allowlist=None):
             self.domain_allowlist = allowlist
 
     def __call__(self, value):
-        # The maximum length of an email is 320 characters per RFC 3696
+        # The maximum length of an email is 254 characters per RFC 5321/5322
```

Comment:
This should not refer **section 3** of RFC 5321.

Reply:
@felixxm did you mean "should not"?

Reply:
> @felixxm did you mean "should not"?

Yes, currently it says `per RFC 5321/5322 section 3`. There is nothing related in this section.

Reply:
Thanks for the clarification!